spin-up-hub/src/store.js

```diff
@@ -115,22 +115,25 @@ const store = createStore({
       })
     },
     async loadSearchData(context) {
-      let data = await fetch(import.meta.env.VITE_API_HOST + "/static/hub-index-data.json")
-      let documents = await data.json()
+      let data = await fetch(import.meta.env.VITE_API_HOST + "/static/hub-index-data.json");
```

Comment:
nit: I would not add semicolons here just to be consistent with the rest of the code.